### PR TITLE
Finish up the builder's instructions

### PR DESCRIPTION
**Pull request by CodaFi**
Instructions that are missing are the exception-handling ones.  We've had a discussion off of here about the difficulty of implementing a sensible interface for them.  My first attempt at this lives in #71.

### CODE REVIEW

**Review by harlanhaskins — COMMENTED** [review]
Tests? Just sanity checking.

**Comment by harlanhaskins** [comment]
Otherwise LGTM!

File: Sources/LLVM/IRBuilder.swift [hunk 1]
```diff
@@ -944,6 +1078,21 @@ public class IRBuilder {
     }
   }
 
+  /// Build a `va_arg` instruction to access arguments passed through the 
+  /// "variable argument" area of a function call. 
+  ///
+  /// This instruction is used to implement the `va_arg` macro in C.
+  ///
+  /// - parameter list: A value of type `va_list*`
+  /// - parameter type: THe type of values in the variable argument area.
```
Comment (harlanhaskins) [inline Sources/LLVM/IRBuilder.swift]: Typo `THe`

**Comment by CodaFi** [comment]
⛵️

**Pull request merged**